test/Microsoft.Azure.Devices.Edge.Test/PriorityQueues.cs

```diff
@@ -33,7 +33,7 @@ public class PriorityQueues : SasManualProvisioningFixture
         const string DefaultLoadGenTestStartDelay = "00:00:20";
 
         [Test]
-        [Category("FlakyOnArm")]
+        [Category("Flaky")]
```

Comment:
I had approved a duplicate PR : https://github.com/Azure/iotedge/pull/5650. But i'll approve this too :)